src/lib/modernExtend.ts

```diff
@@ -312,17 +320,32 @@ export function enumLookup(args: EnumLookupArgs): ModernExtend {
         },
     }];
 
-    return {exposes: [expose], fromZigbee, toZigbee, isModernExtend: true};
+    const configure: Configure = async (device, coordinatorEndpoint, logger) => {
```

Comment:
Only add configure to the result when requested, e.g.

```
const result = {exposes: [expose], fromZigbee, toZigbee, isModernExtend: true}
if (configureReporting) {
  result.configure = ...
}
return result;

Reply:
Does this still make sense if we drop `configureSkipRead` ?
- we have a `configureReporting` payload -> we call `setupAttributes()`
- we do not have a `configureReporting` payload -> we call `setupAttributes(...readOnly=true)`

We will always end up with a configure in this case, either to setup reporting or to just read the value ?

Reply:
Fine with that!